applications/addons/settings/bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php if (!defined('APPLICATION')) exit();
+
+require_once(PATH_APPLICATIONS . DS . 'addons' . DS . 'library' . DS . 'functions.render.php');
```

Comment:
What would be a good way to include this as I use it in the controller and the hooks file?